README.md

```diff
@@ -185,6 +185,20 @@ Now I look at my change again.
 If I can add the same functionality in a non-breaking way - for example, make a new internal function for my use - then I would do so and tag v1.15.2 or v1.16.0 depending on what had to change.
 If I cannot make an equivalent non-breaking change, then I would have to make the breaking change and tag v2.0.0.
 
+### Accidental support for an unsupported dependency
```

Comment:
this should be a subsection under accidentially tagging a breaking release no?

Reply:
I think it's a different kind of breaking release, it's a compatibility error and one that is not fixed by simply release a patch.

That said, I could be convinced to nest this under "Accidental breaking releases" but thought it was a different enough to get its own section.

Reply:
I think it should be immidately after it.
Even if not nested within it

Reply:
ok